ledger/participant-state/kvutils/tools/README.md

```diff
@@ -0,0 +1,22 @@
+# `integrity-check`
+
+A tool that checks the integrity of a ledger dump stored in a file. It can be used to
+verify that a dump made with an old version of the software, can still be successfully processed.
+It can also be used to profile DAML interpretation code in a more isolated fashion.
+
+## Build
```

Comment:
I'm not sure why you would ever want to build the tool without running it. I think you can probably just delete this section.